hw1/AccountTest.java

```diff
@@ -0,0 +1,44 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class AccountTest {
+
+    @Test
+    public void add_NegativeAmount_ReturnsFalse() {
+        Account account = new Account(1);
+        assertFalse(account.add(-10));

```

Comment:
Стоит выделять то что проверяется в отдельную переменную. Лучше стараться писать тесты в стиле given -> when -> then, чтобы было проще понять, что дано и что проверяется. Советую прям разделять комментариями, типа
```suggestion
      // given
       ...
       // when 
       ...
       // then
        ...
```
В случае, когда тесты будут более сложными и с использованием моков, это позволит наглядно понять что делает тест и упростит его поддержку. (https://martinfowler.com/bliki/GivenWhenThen.html)